.circleci/config.yml

```diff
@@ -902,3 +909,70 @@ jobs:
       - run: docker run --net host -v /var/run/docker.sock:/var/run/docker.sock -v ~/project:/grace/test-results/grace-results quay.io/influxdb/grace:daily
       - store_artifacts:
           path: ~/project
+  changelog:
+    parameters:
+      nightly:
+        type: boolean
+    machine:
+      image: ubuntu-2004:202010-01
+    steps:
+      - checkout
+      - install_core_deps
+      - run:
+          name: Install git-cliff
+          command: |
+            set -x
+            git clone --depth=1 --branch v0.2.5 https://github.com/orhun/git-cliff
+            cd git-cliff
+            cargo install git-cliff
+      - when:
+          condition: << parameters.nightly >>
+          steps:
+            - run:
+                name: Nightly Changelog
+                command: |
+                  set -x
+                  cat \<< EOF >> env_vars.sh
+                  S3_PATH="dl.influxdata/platform/nightlies/2.0"
+                  CHANGELOG_FILENAME="CHANGELOG.md"
+                  EOF
+      - unless:
+          condition: << parameters.nightly >>
+          steps:
+            - run:
+                name: Release Changelog
+                command: |
+                  set -x
+                  LAST_RELEASE_TAG=$(git describe --abbrev=0 --tags $(git rev-list --tags=v2* --skip=1 --max-count=1))
+
+                  cat \<< EOF >> env_vars.sh
+                  S3_PATH="dl.influxdata/influxdb/releases"
+                  CHANGELOG_FILENAME="CHANGELOG-${LAST_RELEASE_TAG}.md"
+                  EOF
+      - run:
+          name: Changelog
+          command: |
+            set -x
+            source env_vars.sh
+            COMMIT_FILE_PATH="changelog-commit.txt"
+
+            curl -o ${CHANGELOG_FILENAME} ${S3_PATH}/${CHANGELOG_FILENAME}
+            curl -o ${COMMIT_FILE_PATH} ${S3_PATH}/${COMMIT_FILE_PATH}
+
+            LAST_COMMIT=$(cat $COMMIT_FILE_PATH)
+            NEWEST_COMMIT=${CIRCLE_SHA1}
+            TIMESTAMP="$(date -u '+%Y%m%d')"
+            TAG=${TIMESTAMP}
+            if [ -n "${CIRCLE_TAG}" ]; then
+              TAG=${CIRCLE_TAG}
+            fi
+
+            ./scripts/ci/update-changelog.sh \
+              --commit-range "$LAST_COMMIT..$NEWEST_COMMIT" \
+              --prepend ${CHANGELOG_FILENAME} \
```

Comment:
This release changelog should always be a new file right? If so, would it make sense to not use the `--prepend` flag for release changelogs?

Reply:
oh yeah, I suppose that's true. Good catch.